app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/ErrorModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.kks.nimblesurveyjetpackcompose.model
+
+import com.kks.nimblesurveyjetpackcompose.util.extensions.ErrorType
+
+data class ErrorModel(
+    val errorType: ErrorType? = null,
```

Comment:
Perhaps `errorType` should not be nullable? 🤔 
```suggestion
    val errorType: ErrorType,
```

Reply:
Logically `ErrorType` is the type to differentiate between network error and error with message. At first I had `NONE` type. But I don't think it is relevant. No error mean `ErrorType` is `null` isn't it? That's why I made it nullable. It makes more sense for me. What do you think?

Reply:
If there is no error then why return an `ErrorModel` at all? 🤔